soc/arm/st_stm32/stm32wb/Kconfig.defconfig.stm32wb55xx

```diff
@@ -11,4 +11,11 @@ config SOC
 config NUM_IRQS
 	default 63
 
+if DMA_STM32
+
+config DMAMUX_STM32
```

Comment:
Move to soc/arm/st_stm32/stm32wb/Kconfig.defconfig.series

Reply:
keeping dmamux feature for this serie only